hack/quick_install.sh

```diff
@@ -17,11 +17,11 @@ Help() {
    echo
 }
 
-export ISTIO_VERSION=1.20.4
-export KNATIVE_OPERATOR_VERSION=v1.14.5
-export KNATIVE_SERVING_VERSION=1.13.1
+export ISTIO_VERSION=1.23.2
```

Comment:
We are actually on Istion v1.23.0, as it is a minor version update, it should be fine.